### ISSUE TITLE
Does this support Windows Subsystem For Android? 

### ISSUE THREAD

**Issue by happymimimix** [HuskyDG/magic_overlayfs#77]
Does this work on Windows Subsystem For Android? (rooted with magisk)

**Comment by liorbp** [comment]
Yes. just notice there's an issue with the ZIP that was released for v3.2.2.
You would need to compile it yourself, just follow the readme.

Confirmed to work on [MagiskOnWSALocal](https://github.com/LSPosed/MagiskOnWSALocal)
